vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -185,7 +185,8 @@ def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
                             sorted_token_ids: torch.Tensor,
                             expert_ids: torch.Tensor,
                             num_tokens_post_padded: torch.Tensor,
-                            mul_routed_weight: bool, top_k: int, config: dict):
+                            mul_routed_weight: bool, top_k: int,
+                            config: Dict[int, Any]) -> None:
```

Comment:
Actually this is `Dict[str, Any]` (mapping from e.g. "BLOCK_SIZE_M' to int for example but we probably want to keep open the option to map to other types like float in the future). This is the config dict after the batch size lookup has happened :)

Reply:
Oh my bad. Good catch. Fixed!